adapters/signer_extraction_adapter/README.md

```diff
@@ -0,0 +1,31 @@
+# Signer Extraction Adapter (SEA)
+
+## Overview
+
+The Signer Extraction Adapter is utilized to retrieve the signer information of a given transaction. This is purposefully built to allow application developers to retrieve signer information in the case where the default Cosmos SDK signing information is not applicable. 
```

Comment:
```suggestion
The Signer Extraction Adapter is utilized to retrieve the signature information of a given transaction. This is purposefully built to allow application developers to retrieve signer information in the case where the default Cosmos SDK signature information is not applicable. 
```

Is there an example when this might be true?  I'm assuming stuff like EVM compat chains right?

Reply:
yea applies for EVM chains (evmos is moving away from cosmos txs entirely).